app/src/main/res/values/strings.xml

```diff
@@ -11,7 +11,7 @@
     <string name="action_settings">Einstellungen</string>
     <string name="menu_tourbook">Tourenbuch</string>
     <string name="nav_header_desc">Yet Another Climbing Guide</string>
-    <string name="nav_header_title">YACGuide</string>
+    <string name="nav_header_title">YACguide</string>
```

Comment:
@paetz: Would it not better to change it to `YAC Guide`. It fits more to the long name `Yet Another Climbing Guide` where the word `Guide` is capitalized. I agree that `YACGuide` is hard to read.

Reply:
I think about it and may change it in an upcoming change.